src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -315,18 +321,21 @@ void Graph::InitDescriptors() {
     OV_ITT_SCOPE_CHAIN(FIRST_INFERENCE, taskChain, itt::domains::intel_cpu_LT, "InitDescriptors", "Prepare");
 
     for (auto &node : graphNodes) {
-        OV_ITT_SCOPE_NEXT(FIRST_INFERENCE, taskChain, node->profiling.getSupportedDescriptors);
-        DEBUG_LOG("Get supported primitive descriptors for node: ", node->getName());
-        node->getSupportedDescriptors();
-
-        OV_ITT_SCOPE_NEXT(FIRST_INFERENCE, taskChain, node->profiling.initSupportedPrimitiveDescriptors);
-        DEBUG_LOG("Init supported primitive descriptors for node: ", node->getName());
-        node->initSupportedPrimitiveDescriptors();
-
-        OV_ITT_SCOPE_NEXT(FIRST_INFERENCE, taskChain, node->profiling.filterSupportedPrimitiveDescriptors);
-        DEBUG_LOG("Filter supported primitive descriptors for node: ", node->getName());
-        node->filterSupportedPrimitiveDescriptors();
-
+        try {
+            OV_ITT_SCOPE_NEXT(FIRST_INFERENCE, taskChain, node->profiling.getSupportedDescriptors);
+            DEBUG_LOG("Get supported primitive descriptors for node: ", node->getName());
+            node->getSupportedDescriptors();
+
+            OV_ITT_SCOPE_NEXT(FIRST_INFERENCE, taskChain, node->profiling.initSupportedPrimitiveDescriptors);
+            DEBUG_LOG("Init supported primitive descriptors for node: ", node->getName());
+            node->initSupportedPrimitiveDescriptors();
+
+            OV_ITT_SCOPE_NEXT(FIRST_INFERENCE, taskChain, node->profiling.filterSupportedPrimitiveDescriptors);
+            DEBUG_LOG("Filter supported primitive descriptors for node: ", node->getName());
+            node->filterSupportedPrimitiveDescriptors();
+        } catch (const std::exception& e) {
+            OPENVINO_THROW(node, e.what());
+        }
```

Comment:
The same here and below. These stages usually have their own error messages implemented within nodes themself, and additional context would make the error messages rather cumbersome.

Reply:
The same notice